domain/src/test/kotlin/com/antosik/benchproject/domain/popular/movies/usecase/UpdatePopularMovieUseCaseTest.kt

```diff
@@ -0,0 +1,26 @@
+package com.antosik.benchproject.domain.popular.movies.usecase
+
+import com.antosik.benchproject.domain.popular.movies.entity.MovieModel
+import com.antosik.benchproject.domain.popular.movies.repo.PopularMoviesRepository
+import io.mockk.Runs
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.just
+import io.mockk.mockk
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.Test
+
+internal class UpdatePopularMovieUseCaseTest {
+    val popularMovieRepository = mockk<PopularMoviesRepository>()
```

Comment:
i think that more readable is when we use empty line just after class declaration :) (above line nr 14)

Reply:
I agree with it 😄

Reply:
Done 😄